keras/losses.py

```diff
@@ -1885,6 +1885,15 @@ def _ragged_tensor_sparse_categorical_crossentropy(y_true,
       sparse_categorical_crossentropy, from_logits=from_logits, axis=axis)
   return _ragged_tensor_apply_loss(fn, y_true, y_pred, y_pred_extra_dim=True)
 
+def _check_ground_truth_range(y_true):
+  """ Ensures ground truth labels in binary_crossentropy and 
+      binary_focal_crossentropy lie with 0.0 and 1.0
+  """
+  if tf.math.reduce_max(y_true)<= 1.0 and tf.math.reduce_min(y_true)>= 0.0 \
+    and not tf.math.is_nan(tf.math.reduce_sum(y_true)):
+    return False
```

Comment:
Returning false for a passing case is quite weird. Why not just raise error in this method since the method name is `_check_ground_truth_range`